src/app/app.js

```diff
@@ -0,0 +1,10 @@
+import Sources from '../sources/sources.js';
```

Comment:
relative imports may create difficulties when your app starts getting bigger
It worth to configure absolute path for imports

Reply:
create absolute path